src/inputType/PanInput.ts

```diff
@@ -87,49 +84,34 @@ export function useDirection(
 export class PanInput implements IInputType {
 	options: PanInputOption;
 	axes: string[] = [];
-	hammer = null;
 	element: HTMLElement = null;
+	originalCssProps: { [key: string]: string; } = {};
 	protected observer: IInputTypeObserver;
 	protected _direction;
-	private panRecognizer = null;
+	private activeInput: ActiveInput = {
+		start: [],
+		move: [],
+		end: [],
+	};
+	private isEnabled = false;
```

Comment:
Since we are using boolean property names like isHolded, isRightEdge at input types, I thought isEnabled fits that naming which can know boolean properties easily.

Reply:
I respect that, but remember you don't have to reuse old names :)
But, you know, `isEnabled` is acceptable, but `isHolded` looks quite bad for me.
`holding`, or `isHolding`, if you want, would be a nicer approach in this case I think.
Also, `enabled` and `holding` looks 100% boolean value to me.